cmd/content.go

```diff
@@ -177,9 +182,21 @@ func runContentCmd(cmd *cobra.Command, args []string) (err error) {
 		return errors.Wrap(err, "CommitOnBranchV4")
 	}
 
-	if title := viper.GetString("pr-title"); newBranch && title != "" {
+	if title := viper.GetString("pr-title"); (newBranch || viper.GetBool("force-pr")) && title != "" {
 		body := githubv4.String(viper.GetString("pr-body"))
 		log.Infof("opening pull request from %q to %q", branch, baseBranch)
+
+		existingPR, err := client.GetOpenPullRequestV4(owner, repo, branch, baseBranch)
```

Comment:
If we're going to do the work of looking for open PRs in the common case, then I think perhaps we don't need the `force-pr` flag at all, and can instead change the PR creation gate to just `title != ""`. If the user specifies a `--pr-title`, then always check for a PR and create if missing.

Reply:
yes indeed!